### DISCUSSION TITLE
How often are achievements updated?

### DISCUSSION THREAD

**Discussion by ale-lit** [community/community#18323] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! How often are achievements updated in profile?

Yesterday I did a merge of branches, but a new achievement (YOLO) has not yet appeared.

**Comment by andae** [comment]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements

**Reply by ale-lit** [reply]
Thank you, but this link does not answer this question :(

**Reply by Nishant1500** [reply]
I wanna know it too

**Comment by rudSarkar** [comment]
As per the [blogpost](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/), there is no such information given about the achievements update.

**Comment by zynowhere** [comment]
As far as I know, only Sponsor is updated in real time

**Comment by Nishant1500** [comment]
This answer might help you
https://github.com/github-community/community/discussions/18335#discussioncomment-2927554

**Reply by Gus-Victrix** [reply]
This link leads to a 404 not found error in my browser.

**Comment by ale-lit** [comment]
In some cases, the reward is issued instantly. But it is not yet clear on what basis.

**Reply by Nishant1500** [reply]
Yeah, i just noticed it right now lol

**Comment by ale-lit** [comment, marked as answer]
If all conditions are met correctly, the reward appears instantly
